File: src/main/java/aurocosh/divinefavor/common/item/talismans/arrow/ArrowTalismanAntiGravityArrow.java
package aurocosh.divinefavor.common.item.talismans.arrow;

import aurocosh.divinefavor.common.favor.ModFavor;
import aurocosh.divinefavor.common.item.talismans.arrow.base.ArrowOptions;
import aurocosh.divinefavor.common.item.talismans.arrow.base.ArrowType;
import aurocosh.divinefavor.common.item.talismans.arrow.base.GravityType;
import aurocosh.divinefavor.common.item.talismans.arrow.base.ItemArrowTalisman;

import java.awt.*;
import java.util.EnumSet;

public class ArrowTalismanAntiGravityArrow extends ItemArrowTalisman {
    public ArrowTalismanAntiGravityArrow(String name, ModFavor favor, int favorCost, int color, double arrowDamage, EnumSet<ArrowOptions> options, ArrowType arrowType) {
        super(name, favor, favorCost, color, arrowDamage, options, arrowType);
        gravityType = GravityType.ANTIGRAVITY;
    }
}

File: src/main/kotlin/aurocosh/divinefavor/common/item/arrow_talismans/ArrowTalismanAntiGravityArrow.kt
package aurocosh.divinefavor.common.item.arrow_talismans

import aurocosh.divinefavor.common.item.arrow_talismans.base.ArrowOptions
import aurocosh.divinefavor.common.item.arrow_talismans.base.ArrowType
import aurocosh.divinefavor.common.item.arrow_talismans.base.GravityType
import aurocosh.divinefavor.common.item.arrow_talismans.base.ItemArrowTalisman
import aurocosh.divinefavor.common.spirit.base.ModSpirit
import java.awt.Color
import java.util.*

class ArrowTalismanAntiGravityArrow(name: String, spirit: ModSpirit, favorCost: Int, color: Color, arrowDamage: Double, options: EnumSet<ArrowOptions>, arrowType: ArrowType) : ItemArrowTalisman(name, spirit, favorCost, color, arrowDamage, options, arrowType) {
    init {
        gravityType = GravityType.ANTIGRAVITY
    }
}
